tests/Tests/400__ods_dashboard/414__ods_dashboard_api.robot

```diff
@@ -0,0 +1,374 @@
+    Operation Should Be Forbidden
+    Perform Dashboard API Endpoint GET Call   endpoint=${CM_DASHBOARD_ENDPOINT}    token=${ADMIN_TOKEN}
+    # not clear if it should be Allowed or Forbidden..
+    Operation Should Be Forbidden
+    Create A Dummy Secret In Dashboard Namespace
+    Perform Dashboard API Endpoint GET Call   endpoint=${SECRET_DASHBOARD_ENDPOINT}    token=${BASIC_USER_TOKEN}
+    Operation Should Be Forbidden
+    Perform Dashboard API Endpoint GET Call   endpoint=${SECRET_DASHBOARD_ENDPOINT}    token=${ADMIN_TOKEN}
+    Operation Should Be Forbidden
+    Perform Dashboard API Endpoint DELETE Call   endpoint=${SECRET_DASHBOARD_ENDPOINT}    token=${BASIC_USER_TOKEN}
+    Operation Should Be Forbidden
+    Perform Dashboard API Endpoint DELETE Call   endpoint=${SECRET_DASHBOARD_ENDPOINT}    token=${ADMIN_TOKEN}
+    Operation Should Be Forbidden
+    [Teardown]      Delete Dummy Secret
+
+Verify Access To groups-config API Endpoint
+    [Documentation]     Verifies the endpoint "groups-config" works as expected
+    ...                 based on the permissions of the user who query the endpoint
+
+    [Tags]    ODS-XYZ
+    ...       Tier1
+    ...       Security
+    Perform Dashboard API Endpoint GET Call   endpoint=${GROUPS_CONFIG_ENDPOINT}    token=${BASIC_USER_TOKEN}
+    Operation Should Be Forbidden
+    Perform Dashboard API Endpoint GET Call   endpoint=${GROUPS_CONFIG_ENDPOINT}    token=${ADMIN_TOKEN}
+    Operation Should Be Allowed
+
+
+*** Keywords ***
+Endpoint Testing Setup
+    [Documentation]     Fetches a bearer token for both a RHODS admin and basic user
+    Set Library Search Order    SeleniumLibrary
+    RHOSi Setup
+    ${ADMIN_TOKEN}=   Log In As RHODS Admin
+    Set Suite Variable    ${ADMIN_TOKEN}
+    ${BASIC_USER_TOKEN}=   Log In As RHODS Basic User
+    Set Suite Variable    ${BASIC_USER_TOKEN}
+
+Endpoint Testing Teardown
+    [Documentation]     Switches to original OC context
+    RHOSi Teardown
+
+Log In As RHODS Admin
+    [Documentation]     Perfom OC login using a RHODS admin user
+    Launch Dashboard    ocp_user_name=${TEST_USER.USERNAME}    ocp_user_pw=${TEST_USER.PASSWORD}
+    ...    ocp_user_auth_type=${TEST_USER.AUTH_TYPE}    dashboard_url=${ODH_DASHBOARD_URL}    browser=${BROWSER.NAME}
+    ...    browser_options=${BROWSER.OPTIONS}
+    ${oauth_proxy_cookie}=     Get OAuth Cookie
+    Close Browser
+    [Return]    ${oauth_proxy_cookie}
+
+Log In As RHODS Basic User
+    [Documentation]     Perfom OC login using a RHODS basic user
+    Launch Dashboard    ocp_user_name=${TEST_USER_3.USERNAME}    ocp_user_pw=${TEST_USER_3.PASSWORD}
+    ...    ocp_user_auth_type=${TEST_USER.AUTH_TYPE}    dashboard_url=${ODH_DASHBOARD_URL}    browser=${BROWSER.NAME}
+    ...    browser_options=${BROWSER.OPTIONS}
+    ${oauth_proxy_cookie}=     Get OAuth Cookie
+    Close Browser
+    [Return]    ${oauth_proxy_cookie}
+
+Spawn MinimalPython Notebook Server
+    [Documentation]    Suite Setup
```

Comment:
Space between `Minimal Python`

Reply:
I did intentionally, I will add the space